bin/main.rb

```diff
@@ -1,36 +1,104 @@
-#!/usr/bin/env ruby
-# what do you need from player?
+local_dir = File.expand_path(__dir__)
+$LOAD_PATH.unshift(local_dir)
+
+require 'board.rb'
+require 'player.rb'
```

Comment:
- [ ] [OPTIONAL]  Using require_relative for files in your directory instead of require as this is a ruby convention.

Reply:
Thanks for the review.